### PR TITLE
contributing pendulum

### PR DESCRIPTION
**Pull request by ericmjl**
to make this really awesome package available throughout the community!

### CODE REVIEW

File: recipes/pendulum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - tzlocal
+    - pytz
+    - python-dateutil
```
Comment (jakirkham) [inline recipes/pendulum/meta.yaml]: Should be able to drop all of these except `python` and `setuptools`.

**Comment by patricksnape** [comment]
Patch [this](https://github.com/sdispater/pendulum/blob/master/setup.py#L48) in order not to happen on Windows since MSVC doesn't understand that flag.

**Comment by ericmjl** [comment]
@patricksnape my apologies for noobness, and for getting back to this late, but what was the patch that I needed to add to the feedstock? Please let me know, and I'l get it done!

**Comment by patricksnape** [comment]
Don't pass `'-Wno-unused-function'` if on Windows as Visual Studio doesn't understand that flag.

**Comment by ericmjl** [comment]
Got it, thanks @patricksnape!

File: recipes/pendulum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% set md5 = "bcbac6f8f698d56b8fa793764fda0c85" %}
+
+package:
+  name: pendulum
+  version: {{ version }}
+
+source:
+  fn: pendulum-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pendulum/pendulum-{{ version }}.tar.gz
+  md5: {{ md5 }}
```
Comment (jakirkham) [inline recipes/pendulum/meta.yaml]: Maybe should use `sha256` as PyPI provides this via [pypi.org]( https://pypi.org/ ).

File: recipes/pendulum/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - pendulum
+    - pendulum._extensions
+    - pendulum._extensions.tz
+    - pendulum.lang
+    - pendulum.mixins
+    - pendulum.tz
+
+about:
+  home: https://github.com/sdispater/pendulum
+  license: MIT
```
Comment (jakirkham) [inline recipes/pendulum/meta.yaml]: Please add `license_file: LICENSE`. Also can optionally add `license_family: MIT`.

**Pull request closed without merging**